stat: change to full path in token (#9346)
tldr-pages/tldr: pages/common/stat.md

> - Show file properties such as size, permissions, creation and access dates among others:

Reviewer: The following edit can be applied later in another PR:

```suggestion
- Show a specific file properties such as size, permissions, creation and access dates among others:
```

All descriptions in all pages should be standardized.

Author: This and three suggestions below are implemented  in #9438

`stat {{path/to/file}}`

- Same as above but in a more concise way:

`stat -t {{path/to/file}}`

> - Show filesystem information:

Reviewer: The following edit can be applied later in another PR:

```suggestion
- Show a filesystem information:
```

`stat -f {{path/to/file}}`

- Show only octal file permissions:

`stat -c "%a %n" {{path/to/file}}`

> - Show owner and group of the file:

Reviewer: The following edit can be applied later in another PR:

```suggestion
- Show an owner and a group of a specific file:
```

`stat -c "%U %G" {{path/to/file}}`

> - Show the size of the file in bytes:

Reviewer: The following edit can be applied later in another PR:

```suggestion
- Show a size of a specific file in bytes:
```
